compiler/damlc/daml-opts/daml-opts/DA/Daml/Options.hs

```diff
@@ -470,12 +472,14 @@ locateCppPath = do
         -- In a packaged application, the executable is stored directly underneath
         -- the resources directory because the target produces a tarball which
         -- has the executable directly under the top directory.
+        -- We use a tarball to ensure the required dynamic libraries are
+        -- distributed with daml.
         -- See @bazel_tools/packaging/packaging.bzl@.
-      , runfilesPathPrefix = "stackage" </> "hpp-0.6.4" </> "_install" </> "bin"
-        -- @stackage-exe//hpp
-        -- when running as a bazel target, the executable has the same name
-        -- but comes from the stackage target, so the prefix is different
-        -- from that of the dist target.
+      , runfilesPathPrefix = mainWorkspace </> "compiler" </> "damlc"
+        -- //compiler/damlc:hpp-copy
```

Comment:
I'm guessing this means "we're getting `hpp` from this target" but it wasn't clear upon first read. Maybe elaborate a bit. (I realize this is just a rephrasing of the existing comment.)